code/modules/power/cell.dm

```diff
@@ -117,6 +118,10 @@
 	var/turf/T = get_turf(loc)
 	if(charge == 0)
 		return
+
+	for (var/mob/living/simple_animal/pulse_demon/PD in src)
+		PD.exit_to_turf(src)
```

Comment:
Generally paracode doesn't put spaces after for/if commands, rather just a if(var val) or for(var in val) structure. It's also generally prefered to not capitalize variables due to them showing up as defines through VS code, I'd just put pulse_demons_in_obj as the var or something for this